starter-code/css/main.css

```diff
@@ -6,3 +6,60 @@ Green: #00B172
 White: #FFF
 
 */
+
+/* #spotify1 {
+    float:left;
+    margin: 0 20px 0 20px;
+    height: 100px;
+    width: 15%;
+}
+*/
+
+.navbar {
+    position: fixed 0 0 0 0;
+    height: 30px;
+    margin-bottom: 10px;
+    padding-top: 10px;
+    width: 100%;
+  }
+
+.navbar ul {
+    margin: 0 0 0 66%;
+    padding: 0 20px 0 20px;
+    float: left;
+}
+
+.landing {
+    float: clear;
+    width: 100%;
+    height: 800px;
+    margin: 50px 0 0 0;
+    /*background-image: src("images/landing.jpg")*/
```

Comment:
USE RELATIVE PATHS! it has to look like this: 

```
background-image: url("../images/landing.jpg")
```

(you should check syntax doubts in google)